.editorconfig

```diff
@@ -15,9 +15,6 @@ indent_size = 2
 [*.md]
 trim_trailing_whitespace = false
 
-[*.{kt,kts}]
-ktlint_disabled_rules = import-ordering
-
```

Comment:
Removed as they are unsupported anymore:
`[WARNING] Editorconfig property 'ktlint_disabled_rules' is obsolete and is not used by KtLint starting from version 0.49. Remove the property from all '.editorconfig' files.`